Submodels/Parameterizations.py

```
@@ -0,0 +1,775 @@
+        None
+        """    
+
+        # Load in the reference profile shape from the reference_file. 
+        # Assumes coordinates are sorted counter clockwise from TE of the Upper
+        # surface.
+        # Final assumption: profile shape must be provided in input file with unit chord length
+        # Skips the first row of the profile file as it contains the profile name
+        try:
+            reference_coordinates = np.genfromtxt(reference_file, dtype=float, skip_header=1)
+        except FileNotFoundError as err:
+            raise FileNotFoundError(f"The data input file {reference_file} does not exist in the current working directory.") from err
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)